### ISSUE TITLE
How does this bot suppose to work?

### ISSUE THREAD

**Issue by IdanAdar** [hacklanta/hubot-yardmaster#24]
I have a hubot running fine in Slack.  I now want to add the Jenkins functionality provided by this module.

I run it like this: 
```
HUBOT_SLACK_TOKEN=*** HUBOT_JENKINS_URL=https://**** HUBOT_JENKINS_USER=**@***.*** HUBOT_JENKINS_USER_API_KEY=**** ./bin/hubot --adapter slack
```

I check that the bot still works. Then I call any of the commands provided by this module but nothing works...

**Comment by sebinbenjamin** [comment]
You have to specify the name of the bot before the commands. 
For example, `botName jenkins list` shows jenkins jobs to which the HUBOT_JENKINS_USER has permissions to.

**Comment by IdanAdar** [comment]
Yep, and that's what I do whether it is for the built-in commands of this integration.

In my case: `certbot ping` or `certbot jenkins list`.
How can I ensure the connection to Jenkins works?

**Comment by sebinbenjamin** [comment]
You can try these commands

`hubot jenkins auth set {username} {api key} `- Give Hubot your credentials to use when you issue commands.
`hubot jenkins auth` - See what Jenkins username the Hubot has for you.
`hubot jenkins auth clear` - Clear your Jenkins credentials.

Basically, a POST request is sent to the Jenkins URL with the Jenkins username and API key. So you could try it from a different client if hubot doesn't seem to work, just to make sure it's not a Jenkins issue.

[Jenkins Remote Access API's](https://wiki.jenkins-ci.org/display/JENKINS/Remote+access+API)
[Jenkins Scripted Client Authentication](https://wiki.jenkins-ci.org/display/JENKINS/Authenticating+scripted+clients)

You can add logs to `yardmaster.coffee` if you are sure it is not a Jenkins issue.

**Comment by IdanAdar** [comment]
Added the credentials using `certbot jenkins auth set`, but commands still don't work...
Could it perhaps be that the installation is not correct?

I see `"hubot-yardmaster": "^1.0.30"` in package.json.
I also added `"hubot-yardmaster"` to external-package.json.
